lib/datadog/tracing/contrib/sinatra/tracer_middleware.rb

```diff
@@ -65,6 +66,7 @@ def call(env)
                 # since the latter is unaware of what the resource might be
                 # and would fallback to a generic resource name when unset
                 rack_request_span.resource ||= span.resource if rack_request_span
+                rack_request_span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_ROUTE, datadog_route) if datadog_route
```

Comment:
Line 53 above writes `datadog_route` into `TAG_ROUTE_PATH`, are both of the keys supposed to be set?

Reply:
tbh I don't know, `TAG_ROUTE_PATH` [sets](https://github.com/DataDog/dd-trace-rb/blob/b7b99b2c72ca97cb42b7c8b72accb5571e8c4f3f/lib/datadog/tracing/contrib/sinatra/ext.rb#L23) `sinatra.route.path`.

I have the same situation with Grape, it also [sets](https://github.com/DataDog/dd-trace-rb/blob/b7b99b2c72ca97cb42b7c8b72accb5571e8c4f3f/lib/datadog/tracing/contrib/grape/ext.rb#L24) `grape.route.path`, and I added setting of `http.route` to it as well.

Reply:
I am not entirely sure, but I think we need `http.route` to be set for some appsec features, so probably it is worth adding it.